workers/data_refinery_workers/processors/create_quantpendia.py

```diff
@@ -65,30 +67,42 @@ def download_files(job_context: Dict) -> Dict:
     return job_context
 
 
-def create_result_objects(job_context: Dict) -> Dict:
-    """
-    Store and host the result as a ComputationalResult object.
-    """
-    compendia_organism = _get_organisms(job_context['samples']).first()
-    compendia_version = _get_next_compendia_version(compendia_organism)
-
+@utils.cache_keys('metadata', work_dir_key='job_dir')
+def add_metadata(job_context: Dict) -> Dict:
     logger.debug("Writing metadata for quantpendia.",
-                job_id=job_context['job_id'],
-                organism_name=compendia_organism.name,
-                **get_process_stats())
-    _add_metadata(job_context)
+            job_id=job_context['job_id'],
+            **get_process_stats())
+    smashing_utils.write_non_data_files(job_context)
+    shutil.copy("/home/user/README_QUANT.md", job_context["output_dir"] + "/README.md")
+    return job_context
+
+
+@utils.cache_keys('archive_path', work_dir_key='job_dir')
+def make_archive(job_context: Dict):
```

Comment:
Any reason `make_archive` and `add_metadata` no longer are prefixed with a `_` denoting them as private functions of this namespace?

Reply:
because I made them steps in the pipeline, instead of calling them from `create_result_objects`.

Reply:
So? They still aren't being called from outside this namespace.

Reply:
hmm, only the main function `create_quantpendia` is being called from outside the module. Should I prefix all functions except that one with `_`?

Reply:
Yup!!